lib/fluent/log.rb

```diff
@@ -53,9 +53,17 @@ def self.str_to_level(log_level_str)
       end
     end
 
-    def initialize(out=STDERR, level=LEVEL_TRACE, opts={})
-      @out = out
-      @level = level
+    def initialize(logger, opts={})
+      # overwrites logger.level= so that config reloading resets level of Fluentd::Log
+      orig_logger_level_setter = logger.class.public_instance_method(:level=).bind(logger)
+      me = self
+      # DaemonLogger's log level setting has difference of one level from that of this class.
+      # Then level in this class will be self.level + 1.
```

Comment:
I added the comment.
https://github.com/fluent/fluentd/pull/880/commits/3cb29cb9912052e715c0972701923906a4ee973a

This is for that Serverengine logger ```level=``` overwrites fluentd's logger level. (Not fluentd logger ```level=``` overwrites serverengine)